security/security_profiles_operator/spo-understanding.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: ASSEMBLY
+[id="spo-understanding"]
+= Understanding the Security Profiles Operator
+include::_attributes/common-attributes.adoc[]
+:context: spo-understanding
+
```

Comment:
We don't have any content for the "Understanding" page. This would be a great place to describe what the Operator is doing and why a customer may want to use it. 

We can always comment this page out of the distribution and come back to it later as well.

Reply:
could this presentation provide a starting point? https://docs.google.com/presentation/d/1mS46t1-E2ikbsE4Dyos9E1UL_WxvlYldcX_mK6zUQlI/edit#slide=id.g177196636f1_1_1